### ISSUE TITLE
Automatic Version Patching

### ISSUE THREAD

**Issue by gudenau** [roxas75/rxTools#307]
Because of the recent developments in the 3DS scene, we can no longer use the eShop from the home menu with rxTools; but you can with GW, this appears to be because of the version spoofing in GW.

**Comment by sykhro** [comment]
The reason GW users can install and use the latest versions of *tiger* and *mint* while other CFW users get an infinite loading screen is because of the way GW patches the firmware check that every app runs on launch

However, this patch is likely the same reason we cannot use things like sysupdater in GW mode but we can with rxTools

**Comment by gudenau** [comment]
Config option then?

**Comment by urherenow** [comment]
@sykhro No... Can't use sysupdate because of firm launch. Nothing to do with firmware checks.

**Comment by gudenau** [comment]
Has there been any progress by any chance?
